games-action/openclonk-7.0.recipe

```diff
@@ -0,0 +1,127 @@
+	2009  mizipzor
+	2009-2013  Nicolas Hake
+	2013  Oliver Schneider
+	2010  Peewee
+	2001-2011  Peter Wortmann
+	2012  Philipp Kern
+	2009-2010  Richard Gerum
+	2001-2013  Sven Eberhardt
+	2004-2005, 2009-2013  Tobias Zwick"
+LICENSE="ISC"
+SOURCE_URI="http://www.openclonk.org/builds/release/$portVersion/openclonk-$portVersion-src.tar.bz2"
+CHECKSUM_SHA256="bc1a231d72774a7aa8819e54e1f79be27a21b579fb057609398f2aa5700b0732"
+SOURCE_DIR="openclonk-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	openclonk$secondaryArchSuffix = $portVersion
+	app:OpenClonk$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libjpeg_turbo$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libxrandr$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLEW$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libault$secondaryArchSuffix
+	lib:libat_1.0k$secondaryArchSuffix
+	lib:libcairo_gobject$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgthread_2.0$secondaryArchSuffix
+	lib:libixml$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libthreadutil$secondaryArchSuffix
+	lib:libtinyxml$secondaryArchSuffix
+	lib:libupnp$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES"
+	haiku_devel$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libjpeg_turbo$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libxrandr$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libGLEW$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libault$secondaryArchSuffix
+	devel:libat_1.0k$secondaryArchSuffix
+	devel:libcairo_gobject$secondaryArchSuffix
+	devel:libcairo$secondaryArchSuffix
+	devel:libgdk_pixbuf-2.0$secondaryArchSuffix
+	devel:libgio_2.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libgthread_2.0$secondaryArchSuffix
+	devel:libixml$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libpango_1.0$secondaryArchSuffix
+	devel:libpangocairo_1.0$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libthreadutil$secondaryArchSuffix
+	devel:libtinyxml$secondaryArchSuffix
+	devel:libupnp$secondaryArchSuffix
+	devel:libvorbisfile$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+        "
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+        "
```

Comment:
There are spaces here. Please replace them by one tab, and add a blank line before `BUILD()`.
```diff
-        "
+	"
+
 BUILD()
```